### ISSUE TITLE
datetime component without setting default time cause display issue

### ISSUE THREAD

**Issue by beerchyan** [ionic-team/ionic#13449]
**Ionic version:** 
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...** 
[x] bug report
[ ] feature request

**Current behavior:**

when i use **datetime** component without  setting default time, it will display GTM time but save as  localtime when choosen

```      
<ion-col col-4>
    <ion-label>时间日期控件</ion-label>
    <ion-datetime (ionChange)="getTimeTransform()"  cancelText="取消" doneText="确定" displayFormat="YYYY-MM-DD HH:mm" pickerFormat="YYYY-MM-DD HH:mm" [(ngModel)]="datetimeSelected"></ion-datetime>
</ion-col>
```

```
getTimeTransform(){
   this.datetimeSelectedTransform= moment(this.datetimeSelected).format("YYYY-MM-DD HH:mm:ss");
   console.log(this.datetimeSelectedTransform);
}
```

when i select 2017-01-01 00:00:00
log result are 2017-01-01 08:00:00

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

I have marked this as something to look at after v4 has been released. I tried setting the `initialValue` using a date string with the local TZ, which displayed the time correctly but still brought it back in GMT. Setting the ngModel to a date with a local TZ worked perfectly, but had the date/time initially set (obviously), so clearly something is being treated inconsistently.

Thank you for using Ionic
